subsys/net/ip/net_shell.c

```diff
@@ -161,6 +161,9 @@ static void iface_cb(struct net_if *iface, void *user_data)
 #if defined(CONFIG_NET_IPV6)
 	struct net_if_ipv6_prefix *prefix;
 	struct net_if_router *router;
+#endif
```

Comment:
don't you have a dedicated patch for shell + vlan? that part should go there.

Reply:
That patch is for creating "net vlan" command, the changes here just print vlan information when user gives "net iface" command.